src/server.js

```diff
@@ -9,15 +9,15 @@ app.use(cors());
 const __dirname = path.resolve();
 
 app.get("/", (req, res) => {
-  fs.readFile(path.join(__dirname, "/src/score.json"), (err, data) => {
+  fs.readFile(path.join(__dirname, "/public/score.json"), (err, data) => {
```

Comment:
```suggestion
  fs.readFile(path.join(__dirname, "public/score.json"), (err, data) => {
```
Remove leading / from path as we will call `npm start` from the root of the project